src/Apps/Artwork/Components/OtherWorks/ArtworkContexts/ArtworkContextAuction.tsx

```diff
@@ -70,7 +70,7 @@ export const ArtworkContextAuctionFragmentContainer = createFragmentContainer<{
     const OtherAuctionsList = (
       <OtherAuctions>
         {sales.map(sale => {
-          return <AuctionCard sale={sale} />
+          return <AuctionCard sale={sale} key={(sale as any).id} />
```

Comment:
How come `as any` needed for using in the key assignment but not for sale assignment?